modules/google-maps/src/utils.js

```diff
@@ -1,6 +1,8 @@
 /* global document, google */
 import {Deck} from '@deck.gl/core';
 
+const MAX_LATITUDE = 85.05113;
```

Comment:
Consider adding a comment referencing https://en.wikipedia.org/wiki/Web_Mercator_projection#Formulas or similar, this may be confusing to future readers. :)